src/server/controllers/movie.js

```diff
@@ -0,0 +1,31 @@
+const { Prisma } = require("@prisma/client");
+const prisma = require("../utils/prisma");
+
+const getMovies = async (req, res) => {
+  const getMovies = await prisma.movie.findMany();
+  return res.send({ movies: getMovies });
+};
+const createMovie = async (req, res) => {
```

Comment:
Is the server checking that the user is logged in before it allows a movie to be created? Or is it all handled as part of only rendering the movie options if there is a token? That's a smart way around it!

Reply:
The latter, I also clear the localStorage if you go back to login to give a sense of security, same with my Logout button

Reply:
Nice! Good to see different approaches, I've gone with verifying on server side as well as I don't trust users :P